neurodsp/plts/time_series.py

```diff
@@ -47,18 +47,24 @@ def plot_time_series(times, sigs, labels=None, colors=None, ax=None, **kwargs):
 
     ax = check_ax(ax, (15, 3))
 
-    times = repeat(times) if isinstance(times, np.ndarray) else times
```

Comment:
if `times` is a 2d array, then it can be iterated by leaving it as is, without needing the tile in lines 53-54, right?
In this case, this line just needs to be updated to consider dimensionality, such as:
`times = repeat(times) if (isinstance(arr, np.ndarray) and arr.ndim == 1) else times`